spec/models/manageiq/providers/kubernetes/container_manager/scanning/job_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "spec_helper"
+require 'MiqContainerGroup/MiqContainerGroup'
+
+describe ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job do
+  context "SmartState Analysis Methods" do
+    before(:each) do
+      @guid = MiqUUID.new_guid
+      @zone = FactoryGirl.create(:zone)
+
+      MiqServer.stub(:my_server).and_return(
+        FactoryGirl.create(:miq_server, :guid => @guid, :zone => @zone)
+      )
```

Comment:
There is a pattern for this `@guid, @server, @zone = EvmSpecHelper.create_guid_miq_server_zone`, but I think @kbrock recently cleaned this up even better.